### ISSUE TITLE
Reputation doesn't work with loading quests

### ISSUE THREAD

**Issue by Katrix-** [lorddusk/HQM#73]
If you try to load this questline, it will load fine, but you have to choose the reputation again. When it loads it either says something like no reputation value 0, or something like that. Or it just says "An unknown error occurred while attempting to perform this command". When the second one happens, this is also printed in the log.

When loading:
```java
[00:11:09] [Server thread/ERROR]: Couldn't process command: 'hqm load Rep'
java.lang.ArrayIndexOutOfBoundsException: -2147483648
	at java.util.ArrayList.elementData(ArrayList.java:418) ~[?:1.8.0_25]
	at java.util.ArrayList.get(ArrayList.java:431) ~[?:1.8.0_25]
	at hardcorequesting.reputation.Reputation.getMarker(Reputation.java:603) ~[Reputation.class:?]
	at hardcorequesting.parsing.QuestAdapter$ReputationSettingConstructor.constructReuptationSetting(QuestAdapter.java:473) ~[QuestAdapter$ReputationSettingConstructor.class:?]
	at hardcorequesting.parsing.QuestAdapter.postLoad(QuestAdapter.java:1017) ~[QuestAdapter.class:?]
	at hardcorequesting.commands.CommandLoad.handleCommand(CommandLoad.java:62) ~[CommandLoad.class:?]
	at hardcorequesting.commands.CommandHandler.func_71515_b(CommandHandler.java:93) ~[CommandHandler.class:?]
	at net.minecraft.command.CommandHandler.func_71556_a(CommandHandler.java:94) [z.class:?]
	at net.minecraft.network.NetHandlerPlayServer.func_147361_d(NetHandlerPlayServer.java:739) [nh.class:?]
	at net.minecraft.network.NetHandlerPlayServer.func_147354_a(NetHandlerPlayServer.java:718) [nh.class:?]
	at net.minecraft.network.play.client.C01PacketChatMessage.func_148833_a(SourceFile:37) [ir.class:?]
	at net.minecraft.network.play.client.C01PacketChatMessage.func_148833_a(SourceFile:9) [ir.class:?]
	at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:212) [ej.class:?]
	at net.minecraft.network.NetworkSystem.func_151269_c(NetworkSystem.java:165) [nc.class:?]
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:659) [MinecraftServer.class:?]
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:547) [MinecraftServer.class:?]
	at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java:111) [bsx.class:?]
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:427) [MinecraftServer.class:?]
	at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685) [?:?]
```

Questline:
```json
{
  "name": "Rep",
  "quests": [
    {
      "name": "Test1",
      "x": 17,
      "y": 14,
      "tasks": [
        {
          "type": "REPUTATION",
          "reputation": [
            {
              "reputation": 0,
              "upper": 0,
              "inverted": false
            }
          ]
        }
      ]
    },
    {
      "name": "Unnamed1",
      "x": 58,
      "y": 17,
      "prerequisites": [
        "Test1"
      ],
      "tasks": [
        {
          "type": "REPUTATION",
          "reputation": [
            {
              "reputation": 0,
              "lower": 0,
              "upper": 1,
              "inverted": false
            }
          ]
        }
      ]
    }
  ],
  "reputation": [
    {
      "name": "Test",
      "neutral": "Neutral",
      "markers": [
        {
          "name": "Test1",
          "value": 5
        },
        {
          "name": "Test2",
          "value": 10
        }
      ]
    }
  ]
}
```

**Comment by way2muchnoise** [comment]
This might be fixed in a015c1a0727911589fb18d2ae761e7e5b9c78a6f could you test this?

**Comment by Katrix-** [comment]
Nope. Tried making a new set of quest with the same content and everything, but still the same.

**Comment by lorddusk** [comment]
Have you tried with 4.4.2? Because this is fixed in that version ( got released this afternoon )

**Comment by Katrix-** [comment]
Yes, I am completely sure that I was using 4.4.2. I could place the reputation meter anywhere, so definitively 4.4.2. Tested on a completely fresh instance with nothing other than HQM 4.4.2. When trying to load the quest, I get basically the same log.

**Comment by canilsen** [comment]
Having the same problem. I get "Missing reputation value 0" in chat when loading chapters. The reputation group is allready created, and should be identical to what is saved in the chapter's quests.

**Comment by way2muchnoise** [comment]
Closed due to no longer supporting 1.7.10, please check if this problem persists in the latest 1.9.4 version. Saving, loading and networking are rewritten for 1.9+, this should resolve this issue
